MunchkinLevelCounter/UI/Charts/ChartsPresenter.swift

```diff
@@ -26,28 +27,29 @@ class ChartsPresenter: Presenter {
     
     func loadChartData(type: ScoreType) {
         loggingPrint("Loading chart data")
-        disposable = dataManager.getLineData(type: type.rawValue)
+        dataManager.getLineData(type: type.rawValue)
             .subscribeOn(ConcurrentDispatchQueueScheduler(qos: DispatchQoS.background))
             .observeOn(MainScheduler.instance)
             .subscribe(onNext: { lineChartData in
                     self.chartsView?.showPlayersCharts(chartData: lineChartData)
                 }
             )
+            .disposed(by: disposeBag)
     }
     
     func loadPlayers(sortType: ScoreType) {
-        disposable = dataManager.getPlayers(sortType: sortType)
+        dataManager.getPlayers(sortType: sortType)
             .subscribeOn(ConcurrentDispatchQueueScheduler(qos: DispatchQoS.background))
             .observeOn(MainScheduler.instance)
             .subscribe(onNext: { players in
                     self.chartsView?.showPlayersList(players: players)
                 }
             )
+            .disposed(by: disposeBag)
     }
     
     func detachView() {
         chartsView = nil
-        disposable?.dispose()
```

Comment:
>Dispose bags are used to return ARC like behavior to RX. When a DisposeBag is deallocated, it will call dispose on each of the added disposables.

As I understood, when you have something like this:
```swift
class SomeClass {
    let bag = DisposeBag()
    func doSomething() {
        ...
        .disposed(by: bag)
    }
}
```
Then, when the bag’s `deinit` is called (when the `SomeClass` object gets deallocated) the `Disposables` that didn’t finish will be disposed of.